packages/react/src/Link/Link.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { ChevronRight } from '@amsterdam/design-system-react-icons'
+import clsx from 'clsx'
+import { AnchorHTMLAttributes, ForwardedRef, forwardRef, PropsWithChildren } from 'react'
+import { Icon } from '../Icon/Icon'
+
+export type LinkOnBackground = 'default' | 'light' | 'dark'
+export type LinkVariant = 'standalone' | 'standaloneBold' | 'inline' | 'inList'
```

Comment:
Do we want to export these types? I can't imagine why we would want to use them outside this component. I think we can define directly in `LinkProps`

Reply:
Wanted to prevent the inability Utrecht has to import component props. But we'll fix that if it comes around.